server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/UnsignedInt.scala

```diff
@@ -19,50 +19,11 @@
 
 package org.apache.james.jmap.model
 
-import com.google.common.math.LongMath
-import org.slf4j.{Logger, LoggerFactory}
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Interval.Closed
 
 object UnsignedInt {
-  private val LOGGER: Logger = LoggerFactory.getLogger(classOf[UnsignedInt])
-  private val VALIDATION_MESSAGE = "value should be positive and less than 2^53"
+//Unsigned int between [0, 2^53]
+type UnsignedInt = Long Refined Closed[0L, 9007199254740992L]
```

Comment:
`2L << 53` would not work?

Reply:
No your expression returns `18014398509481984` not `2^53 = 9007199254740992`

Reply:
on a side note : 
i try doing something like : 
```
val MAX_VALUE = LongMath.pow(2,53)
type UnsignedInt = Long Refined Closed[0L, MAX_VALUE.type]
```

it worked for simple case and in the worksheed but failed in some macro expansion :/

Reply:
he probably meant 1L<<53 :)

Reply:
It is the good value but cannot be used here